backend/server.js

```diff
@@ -27,7 +27,7 @@ app.use('/', express.static('public'));
 app.post('/fiatPayment', async (request, response, next) => {
     try {
         const fiatPaymentData = await getFiatData();
-        fiatPaymentData.shopper = "5c3e090cfd77ee6054c03883";  // Hard-coded shopper ID
+        fiatPaymentData.shopper = "5c3e0757c67b1a3deadb855a";  // Hard-coded shopper ID
```

Comment:
What you think to export shopper id to config.json?

Reply:
I've added the ID to the config